src/sdk/javascript/examples/functions/index.js

```diff
@@ -83,17 +85,24 @@ class IntegratedFunction extends FunctionTalent {
 
     getRules() {
         return new AndRules([
-            new Rule(new OpConstraint(`${this.id}.trigger`, OpConstraint.OPS.GREATER_THAN, 20, DEFAULT_TYPE, VALUE_TYPE_RAW)),
-            new Rule(new ChangeConstraint(`${this.id}.trigger`, DEFAULT_TYPE, VALUE_TYPE_RAW))
+            new Rule(new OpConstraint(`${this.id}.trigger`, OpConstraint.OPS.LESS_THAN_EQUAL, 15, DEFAULT_TYPE, VALUE_TYPE_RAW)),
+            //new Rule(new ChangeConstraint(`${this.id}.trigger`, DEFAULT_TYPE, VALUE_TYPE_RAW))
```

Comment:
Is this commented code still needed?

Reply:
Yes, I will uncomment it, to make the example show it's glory ;)